### DISCUSSION TITLE
Add translations to all 'collection field' fields containing a string

### DISCUSSION THREAD

**Discussion by paulboudewijn** [directus/directus#9902] [Feature Requests]
Some of my users use the app in English, others in Dutch. The app itself is nicely multilingual, but not all the configurable strings are. 
For example, when configuring a collection field, only translations for 'field name' can be entered. It would be nice if this would also be possible for:

- notes (duplicate of #4477)
- interface placeholders, labels, etc.
- dropdown choices
- display templates

So practically everywhere a string value is configured.

**Comment by azrikahar** [comment]
> so practically everywhere a string value is configured.

This will be possible once #12170 gets merged 🙌

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
